project_timesheet_time_control/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==============================
+Project Timesheet Time Control
+==============================
+
+* This module adds a button at account analytic line level to compute the spent
+  time, in minutes, from start date to the current moment.
+* It adds a datetime field that replaces ``date`` field in tree view, and write
+  date field with datetime field value.
+* It also adds a task filter and a group by task options in the search.
+* Finally, it allows to open and close tasks from account analytic lines.
+  The selected closed stage is the first one that is found with the mark
+  "Closed" checked.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
The current text talks about "account analytic line", but end users will be more familiar with "Project Tasks" and "Timesheet Lines", so I find it unclear as is.
It would help to add here a list of steps for the end user that show the typical use case.